gauntlet/packages/gauntlet-solana-contracts/src/lib/utils.ts

```diff
@@ -7,3 +11,34 @@ export const divideIntoChunks = (arr: Array<any> | Buffer, chunkSize: number): a
   }
   return chunks
 }
+
+export const parseContractErrors = async (sendTx: Promise<String>, idl: Idl): Promise<string> => {
+  let txhash
+  try {
+    txhash = await sendTx
```

Comment:
You can just return here. If succeeds, it will return, it will throw otherwise
```suggestion
    return await sendTx
```